mava/core.py

```diff
@@ -189,4 +190,4 @@ def trainer(
 
     @abc.abstractmethod
     def distributor(self) -> Any:
```

Comment:
We can remove this since it is no longer used. But I will do it in another PR. :)

Reply:
Sounds good :slightly_smiling_face: